README.md

```diff
@@ -155,6 +155,10 @@ new features and changes.
 * [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-cli)
 * [Kubernetes Community site](http://kubernetes.io/community/)
 
+### Contribution
+
+Interested in contributing to Krew? Please refer to our [Contributing Guidelines](https://github.com/kubernetes-sigs/krew/blob/master/CONTRIBUTING.md) for more details
```

Comment:
can you please follow the link convention at `## Documentation` section above,
also can you please do 80-character line wraps?

Reply:
I have changed it as per the link convention

Reply:
looks like you missed the 80-character line wrap part :)

Reply:
somehow it didn't reflect in the changes. I have wrapped it now. :)